pkg/eventpersister/persister/persister.go

```diff
@@ -645,3 +662,25 @@ func (p *Persister) createUserEvent(
 	eventStorage := v2ec.NewEventStorage(p.postgresClient)
 	return eventStorage.CreateUserEvent(p.ctx, event, id, environmentNamespace)
 }
+
+func (p *Persister) upsertEvaluationCount(event proto.Message, environmentNamespace string) error {
+	if e, ok := event.(*eventproto.EvaluationEvent); ok {
+		eck := p.key(eventCountKey, e.FeatureId, e.VariationId, e.Timestamp)
+		_, err := p.evaluationCountCacher.Increment(eck)
+		if err != nil {
+			return err
+		}
+		uck := p.key(userCountKey, e.FeatureId, e.VariationId, e.Timestamp)
+		_, err = p.evaluationCountCacher.PFAdd(uck, []string{e.UserId})
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (p *Persister) key(kind, featureID, variationID string, timestamp int64) string {
```

Comment:
Sorry for mentioning this now, but we also need the `environmentNamespace` in the key.

- Event count: `ec:environment_namespace:feature_flag_id:variation_id:daily_timestamp`
- User count: `uc:environment_namespace:feature_flag_id:variation_id:daily_timestamp`